src/modules/home.js

```diff
@@ -0,0 +1,39 @@
+function setSwap(id) {
```

Comment:
## Aim to be `DRY`
- [ ] This functionality has already been defined in the `contacts` module. How about we use the principle of `DRY` and employ the `Write Once Run Everywhere` functionality since we aim to have a modularized codebase. it would be ideal to have this function imported in there that to redefine it.
- [ ] Do this for the remainder of the modules and convert all the function declarations to `ES6` syntax. Thank you!